packages/synthetix-main/contracts/modules/core/AccountModule.sol

```diff
@@ -23,70 +25,15 @@ contract AccountModule is IAccountModule, OwnableMixin, AccountRBACMixin, Initia
     error OnlyTokenProxyAllowed(address origin);
     error InvalidRole();
 
-    // ---------------------------------------
-    // Chores
-    // ---------------------------------------
-    function _isInitialized() internal view override returns (bool) {
-        return _accountModuleStore().initialized;
-    }
-
-    function isAccountModuleInitialized() external view override returns (bool) {
-        return _isInitialized();
-    }
-
-    function initializeAccountModule() external override onlyOwner onlyIfNotInitialized {
-        AccountModuleStore storage store = _accountModuleStore();
-
-        AccountToken firstImplementation = new AccountToken();
-
-        UUPSProxy accountProxy = new UUPSProxy(address(firstImplementation));
-
-        address accountProxyAddress = address(accountProxy);
-        AccountToken account = AccountToken(accountProxyAddress);
-
-        account.nominateNewOwner(address(this));
-        account.acceptOwnership();
-        account.initialize("Synthetix Account", "synthethixAccount", "", address(this));
-
-        store.account = Satellite({
-            name: "synthethixAccount",
-            contractName: "AccountToken",
-            deployedAddress: accountProxyAddress
-        });
-
-        store.initialized = true;
-
-        emit AccountCreated(accountProxyAddress);
-    }
-
-    function _getSatellites() internal view override returns (Satellite[] memory) {
-        Satellite[] memory satellites = new Satellite[](1);
-        satellites[0] = _accountModuleStore().account;
-        return satellites;
-    }
-
-    function getAccountModuleSatellites() public view override returns (Satellite[] memory) {
-        return _getSatellites();
-    }
-
-    function upgradeAccountImplementation(address newAccountTokenImplementation)
-        external
-        override
-        onlyOwner
-        onlyIfInitialized
-    {
-        AccountToken(getAccountAddress()).upgradeTo(newAccountTokenImplementation);
-    }
-
-    function getAccountAddress() public view override returns (address) {
-        return _accountModuleStore().account.deployedAddress;
+    function getAccountAddress() public view override returns (INftModule) {
+        return _getNft(ACCOUNT_SYSTEM);
```

Comment:
AccountTokenModule originaly created a Satellite (AccountToken) that is a specialized NFT that will use the transferAccount hook. 

I didn't find that hook call in the AccountTokenModule (maybe I'm missing something).